recipes/tlsh/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set name = "tlsh" %}
+{% set version = "3.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/trendmicro/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 09e116130d98d6ed180624ff9f7da22b669ccd6f24e316f59019bf56436ee444
+  patches:
+    - make.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - echo tlsh metapackage has no tests
+
+outputs:
+  - name: libtlsh
+    script: install-libtlsh.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('libtlsh', max_pin='x.x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make
+    test:
+      commands:
+        - echo libtlsh tested are during build
+  - name: python-tlsh
+    script: install-python-tlsh.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('python-tlsh', max_pin='x.x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make
+      host:
+        - {{ pin_subpackage('libtlsh', exact=True) }}
+        - python
+        - pip
+      run:
+        - {{ pin_subpackage('libtlsh', exact=True) }}
+        - python
+    test:
+      source_files:
+        - Testing
+        - py_ext
+      imports:
+        - tlsh
+      commands:
+        - cd Testing && sh python_test.sh
+  - name: tlsh-tools
+    script: install-tlsh-tools.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('libtlsh', max_pin='x.x') }}
+    test:
+      commands:
+        - tlsh --help
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make
+      run:
+        - {{ pin_subpackage('libtlsh', max_pin='x.x') }}
```

IIRC You have to state the name of the top level package as well in multi output recipes:
```suggestion
        - {{ pin_subpackage('libtlsh', max_pin='x.x') }}
  - name: {{ name|lower }}
```

---

I've been basing most of this off:

https://github.com/conda-forge/lief-feedstock/blob/master/recipe/meta.yaml

I don't see `lief` called out explicitly, and don't know where it would go. The docs discuss metapackage stuff: are you suggesting a `tlsh` metapackage that depends on all three of the subpackages?